pc-agent/src/api/websocket_server.py

```
@@ -423,10 +439,9 @@ async def handle_websocket_messages(websocket: WebSocket, device_id: str, device
                 await self._handle_voice_command(websocket, message_data, device_info)
             elif message_type == "ping":
                 # Handle ping for connection health check
-                await self._send_message(websocket, {
-                    "type": "pong",
-                    "timestamp": get_current_timestamp()
-                })
+                await self._send_message(
```

Ruff (0.14.5)

Undefined name `self`

(F821)